cluster/node-pools/worker-karpenter/provisioners.yaml

```diff
@@ -99,20 +99,25 @@ spec:
   userData: |
 {{.UserData | indent 4}}
   tags:
-    InfrastructureComponent: "true"
-    # TODO: maybe use application label on dedicated node pools?
-    application: kubernetes
-    component: worker
-    environment: "{{ .Cluster.Environment }}"
```

Comment:
are application/component tags gonna be set in another way? The TODO was an idea to improve the data in the future, but we should ensure we have application and component labels on instances.